internal/ingress/controller/config/config.go

```diff
@@ -715,6 +715,7 @@ func NewDefault() Configuration {
 			LimitRate:                0,
 			LimitRateAfter:           0,
 			ProxyBuffering:           "off",
+			ProxyMaxTempFileSize:     "1024m",
```

Comment:
`1024m` is the default value http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_max_temp_file_size